ios/AppLovinMAX.m

```diff
@@ -142,8 +144,12 @@ - (instancetype)init
         }
     }
     
+    ALSdkSettings *settings = [[ALSdkSettings alloc] init];
+    settings.consentFlowSettings.enabled = self.consentFlowEnabledToSet.boolValue ?: NO;
+    settings.consentFlowSettings.privacyPolicyURL = self.privacyPolicyURLToSet;
```

Comment:
Should we nil out `self.consentFlowEnabledToSet` and `self.privacyPolicyURLToSet` for consistency's sake?

Reply:
Also, 

```
settings.consentFlowSettings.enabled = self.consentFlowEnabledToSet.boolValue ?: NO;
```

can technically just be 

```
settings.consentFlowSettings.enabled = self.consentFlowEnabledToSet.boolValue;
```